compiler/bindings/python/iree/build/onnx_actions.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from iree.build.executor import BuildAction, BuildContext, BuildFile, BuildFileLike
+
+__all__ = [
+    "onnx_import",
+]
+
+
+def onnx_import(
```

Comment:
Will we end up with similar actions for other frameworks? TFLite should be similarly straightforward with self-contained input artifacts (`.tflite` / `.onnx`). Similar story for standalone StableHLO (already exported from TF / JAX / etc.), but I'm curious how PyTorch programs like those imported through iree-turbine will integrate with this new package.

Reply:
There will be some kind of import module that we can fork to a subprocess, run and consume the result.